### PR TITLE
chore(llc,core,ui,persistence,localization): 3.0.0

### PR DESCRIPTION
**Pull request by imtoori**

### CODE REVIEW

File: packages/stream_chat_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-## Upcoming
+## 3.0.0
+🛑️ Breaking Changes from `2.2.1`
```
Comment (HayesGordon) [inline packages/stream_chat_flutter/CHANGELOG.md:2]: New line

**Review by HayesGordon — COMMENTED** [review]
LGTM. We should be more consistent with our use of full-stops in the CHANGELOG.md - sometimes we end with one sometimes not. We can also consider specifying in the changelogs for the other packages that we updated our dependency on `stream_chat` to version 3 with a link to that changelog.

File: packages/stream_chat_flutter_core/CHANGELOG.md [hunk 2]
```diff
@@ -27,6 +27,8 @@
     limit = 30
     ```
 
+- `UserListViewCore` `filter` property now is non-nullable.
```
[the remark this answers is not in the document]
Comment (imtoori) [reply packages/stream_chat_flutter_core/CHANGELOG.md:30]:
```suggestion
- `UserListCore` `filter` property now is non-nullable.
```

File: packages/stream_chat_flutter_core/CHANGELOG.md [hunk 3]
```diff
@@ -27,6 +27,8 @@
...
+
 🔄 Changed
 
 - `UserListViewCore` filter property now has a default value.
```
[the remark this answers is not in the document]
Comment (imtoori) [reply packages/stream_chat_flutter_core/CHANGELOG.md:34]:
```suggestion
- `UserListCore` filter property now has a default value.
```

File: packages/stream_chat_flutter/CHANGELOG.md (hunk 1 above)
Comment (imtoori) [reply packages/stream_chat_flutter/CHANGELOG.md:2]:
```suggestion

🛑️ Breaking Changes from `2.2.1`
```

**Review by HayesGordon — APPROVED** [review]
LGTM

**Pull request merged**